change.md

```diff
@@ -0,0 +1,5 @@
+Change
```

Comment:
This file is redundant and unncessary (this information is all included in the git log).  Feel free to keep notes locally but do not include them in your commits.  Any information about why files were added or changes should instead be included in the commit message which introduces that change.